src/app/main/Plugin.tsx

```diff
@@ -20,6 +24,7 @@ export class Plugin extends React.Component<IPluginProps, any> {
     super(props)
 
     this.onLabelLinkClicked = this.onLabelLinkClicked.bind(this)
+    this.onError = this.onError.bind(this)
```

Comment:
To avoid binding the component `this` you can declare your onError method differently. See how I do it [here](https://github.com/Ethan-Arrowood/camper-tracking-app/blob/2c69f92fcd3a6116c9c2ebab1a7ead68a9150817/client/src/pages/CamperList/CamperList.js#L37-L43)

Also check out the next comment on the `onError` function where I show you the proper rewrite.

Reply:
makes sense, but punting for now as it requires a larger change across the codebase for consistency: https://github.com/bengreenier/overlayed/issues/37